dts/arm/st/stm32l0.dtsi

```diff
@@ -80,6 +80,15 @@
 			label = "UART_2";
 		};
 
+		lpuart1: serial@40004800 {
+			compatible = "st,stm32-lpuart";
```

Comment:
The compatible should probably be `"st,stm32-lpuart", "st,stm32-uart"` as we are using the common driver for all UART flavours. I checked the Linux binding for STM32, but it does not have support for LPUART.

Reply:
I'll fix that.